src/en-us.strings.js

```diff
@@ -190,5 +190,18 @@ export default {
 	'pdfReader.convertToHighlight': 'Convert to Highlight',
 	'pdfReader.convertToUnderline': 'Convert to Underline',
 	'pdfReader.size': 'Size',
-	'pdfReader.merge': 'Merge'
+	'pdfReader.merge': 'Merge',
+    'pdfReader.a11yAnnotationModifierMac': 'Option',
+    'pdfReader.a11yAnnotationModifier': 'Alt',
+    'pdfReader.a11yAnnotationModifierControl': 'Control',
+    'pdfReader.a11yTextualAnnotationFindInDocumentInstruction': 'To turn a search result into a highlight or underline annotation, press',
+    'pdfReader.a11yTextualAnnotationInstruction': 'To annotate text via the keyboard, first use “Find in Document” to locate the phrase. Then, to turn the search result into an annotation, press Control -',
+    'pdfReader.a11yAnnotationInstruction': 'To add this annotation to the document, focus the document and press Control -',
+    'pdfReader.a11yAnnotationNotSupported': 'This annotation type cannot be created via the keyboard.',
+    'pdfReader.a11yMoveAnnotation': 'Use the arrow keys to move the annotation.',
+    'pdfReader.a11yEditTextAnnotation': 'To move the end of the text annotation, use the left/right arrow keys while holding Shift. To move the start of the annotation, use the arrow keys while holding Shift -',
+    'pdfReader.a11yResizeAnnotation': 'To resize the annotation, use the arrow keys while holding Shift.',
+    'pdfReader.a11yAnnotationSelected': 'Annotation selected',
+    'pdfReader.a11yAnnotationPopupAppeared': 'Use Tab to navigate the annotation popup.'
```

Comment:
Is this last one necessary? Is the concern that someone wouldn't know that there's a popup, and so we have to tell them how to navigate it? Because Tab is obviously how you navigate everything.

Reply:
> Is the concern that someone wouldn't know that there's a popup, and so we have to tell them how to navigate it?

That's exactly it. I remember we discussed earlier that currently there is no way to know that something happened when the popup opens. So this is just a way of saying that a popup to edit the annotations appeared and it is the next focusable element. There very much might be a better way to phrase it!

Reply:
No, that makes sense.